docs/markdown/snowplow_event_splitting_overview.md

```diff
@@ -0,0 +1,76 @@
+{% docs __snowplow_event_splitting__ %}
+
+{% raw %}
+
+# Snowplow Event Splitting Package
+
+Welcome to the model documentation site for the Snowplow Event Splitting dbt package. The package provides 2 macros and a python script that is used to generate your models; these models provide a table per event type that you specify, and can also produce a thin table of the split events and a user table, for use within downstream ETL tools. 
+
+**For more information, including a detailed operation guide please visit the [Snowplow Docs](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/).**
+
+*Note this model design doc site is linked to latest release of the package. If you are not using the latest release, [generate and serve](https://docs.getdbt.com/reference/commands/cmd-docs#dbt-docs-serve) the doc site locally for accurate documentation.*
+
+## Overview
+
+This package consists of two macros, a python script, and some example configuration files to help you get started:
+
+  - `split_events` _(macro)_: This macro does the heavy lifting of the package, taking a series of inputs to generate the SQL required to split the events table and flatten (1 level) of any [self-describing event][self-desc-events] or [context][sp-contexts] columns. While you can use this macro manually it is recommended to create the models that use it by using the script provided.
+
+  - `users_table` _(macro)_: This macro takes a series of inputs to generate the SQL that will produce your users table (1 row per user, with the latest values of the contexts you specify as the other columns), using the `user_id` column and any custom contexts from your events table.
```

Comment:
Might be worth mentioning here that latest values is based on `collector_tstamp`

Reply:
Agree, going to match this with the new bits to the docs site, but going to do it a new branch along with the repo name change to keep it all a bit cleaner.